pkgs/development/libraries/science/math/magma/generic.nix

```diff
@@ -115,14 +117,26 @@ stdenv.mkDerivation {
     "test"
   ];
 
+  patches = lib.optionals (version == "2.9.0") [
```

Comment:
Doesn't actually work because we're using release tarballs with prebuilt hipified code https://github.com/icl-utk-edu/magma/pull/27#issuecomment-2689205794

Reply:
Adding this cmakeFlag and ditching the patch should work for now.

```nix
(strings.cmakeFeature "ROCM_CORE" "${effectiveRocmPackages.clr}")
```

If the PR gets merged we can remove that line after the next release.